docs/examples/fmi_ls_bus_manifest_example.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fmiLayeredStandardManifest
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../schema/fmi3LayeredStandardBusManifest.xsd"
+	xmlns:fmi-ls="http://fmi-standard.org/fmi-ls-manifest"
+	fmi-ls:fmi-ls-name="org.fmi-standard.fmi-ls-bus"
+	fmi-ls:fmi-ls-version="1.0.0"
+	fmi-ls:fmi-ls-description="Layered Standard for Bus"/>
```

Comment:
I would prefer "Layered Standard for Network Communication" as the official name of the LS...
However, this is a description field and is probably displayed in the importing tool. Therefore, it should provide a little more information about the contents of this layered standard. I will create an issue for that here and also in the fmi-ls-xcp repo.